CHANGELOG.latest.md

```diff
@@ -1 +1,8 @@
-Initial v6.0.0 alpha release
+### New Features
+* [CF-1119] - Default offer now uses best offer logic (longest free, best savings, base plan) (#754) via Josh Holtz (@joshdholtz)
+* [CF-1147] Use GoogleProrationMode in purchases (#752) via beylmk (@beylmk)
+* [CF-1148] Add proration mode (#751) via beylmk (@beylmk)
+### Other Changes
+* [CF1120] - Update SDK docs for default offer logic (#762) via Josh Holtz (@joshdholtz)
+* Clean up Google-related TODOs (#736) via beylmk (@beylmk)
+* [CF-1135] Remove requirement for `StoreProduct` when purchasing a `PurchaseOption` (#739) via Josh Holtz (@joshdholtz)
```

Comment:
NABD, but i'd call that one a new feature!

Reply:
Oh yeah! Looks like I should probably make sure I give PRs proper tags 🤷‍♂️

Reply:
oooo i forgot these are auto-generated now 😻